docs/06-concepts/18-testing/05-caveates-and-edge-cases.md

```diff
@@ -0,0 +1,35 @@
+# Caveats
+
+## Transactions
+
+As explained in [the basics](the-basics#seeding-the-database), the test tools are by default wrapping each test case in a transaction. That ensures  database calls and even further transactions are contained within that top level transaction. Since nested transactions are not natively supported by PostgreSQL, the test tools emulates this behaviour by using save points.
+
+The following two cases are deviating from true transaction behavior:
+
+### Concurrent transactions
```

Comment:
Question: We have taken a different pattern for the database exceptions being quelled compared to concurrent transactions. One is documented here and one is documented in "the basics". Should we document them the same way?